apps/condo/domains/ticket/components/TicketForm/TicketFormContext.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+
+import { TicketOrganizationSetting } from '@app/condo/schema'
+import { TicketOrganizationSetting as TicketSetting } from '@condo/domains/ticket/utils/clientSchema'
+
+type TicketFormContextType = {
+    ticketSetting: TicketOrganizationSetting,
+    isAutoDetectedDeadlineValue: boolean,
+    setIsAutoDetectedDeadlineValue: React.Dispatch<React.SetStateAction<boolean>>,
+    ticketSettingLoading: boolean,
+    isExistedTicket: boolean
+}
+
+export const TicketFormContext = React.createContext<TicketFormContextType>(null)
+
+export const useTicketFormContext = (): TicketFormContextType => React.useContext(TicketFormContext)
+
+export const TicketFormContextProvider = ({ children, organizationId, isExistedTicket }) => {
+    const [isAutoDetectedDeadlineValue, setIsAutoDetectedDeadlineValue] = useState<boolean>(true)
+
+    const { obj: ticketSetting, loading: ticketSettingLoading } = TicketSetting.useObject({
```

Comment:
What if ticketSetting === undefinded

Reply:
In the absence of a ticket, a default deadline is set - 8 days